en/chronos.rst

```diff
@@ -203,20 +219,48 @@ objects::
     echo $date;
 
     // Different standard formats
-    echo $date->toAtomString();
-    echo $date->toCookieString();
-    echo $date->toIso8601String();
-    echo $date->toRfc822String();
-    echo $date->toRfc850String();
-    echo $date->toRfc1036String();
-    echo $date->toRfc1123String();
-    echo $date->toRfc2822String();
+    echo $dt->toAtomString();      // 1975-12-25T14:15:16-05:00
```

Comment:
I don't want to bother to much with that but all example in the page so far used ``$date`` instead of ``$dt``